reference/5.1/Microsoft.PowerShell.Core/About/about_Comparison_Operators.md

```diff
@@ -61,10 +61,11 @@ which always return a Boolean value.
 
 Windows PowerShell supports the following comparison operators.
 
--eq
-Description: Equal to. Includes an identical value.
-Example:
+-eq  
```

Comment:
This should be section header (for example see **Catch** in about_Language_Keywords.md for example)